azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -611,12 +620,8 @@ public void shutdown() {
    */
   private void shutdownInternal() {
     getFlowRunnerManager().shutdown();
-    try {
-      shutdownNow();
-      logger.warn("Shutdown AzkabanExecutorServer complete");
-    } catch (Exception e) {
-      logger.error(e);
-    }
+    // trigger shutdown hook
+    System.exit(0);
```

Comment:
Why is the shutdownNow call being skipped here?
How is the web server being killed ?